app/models/setting/content.rb

```diff
@@ -23,6 +23,7 @@ def self.load_defaults
         self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."), true),
         self.set('force_post_sync_actions', N_("Force post sync actions such as indexing and email even if no content was available."), false),
         self.set('default_download_policy', N_("Default download policy for repositories (either 'immediate', 'on_demand', or 'background')"), "immediate"),
+        self.set('default_proxy_download_policy', N_("Default download policy for Smart Proxy syncs (either 'inherit', 'immediate', 'on_demand', or 'background')"), "on_demand"),
```

Comment:
Can this setting be a dropdown box? something like https://github.com/theforeman/foreman_discovery/blob/develop/app/models/setting/discovered.rb#L33 or app/helpers/katello/concerns/settings_helper_extensions.rb maybe?